### PR TITLE
Feature add telecareclients

### PR DESCRIPTION
**Pull request by vijivasanthan**

### CODE REVIEW

**Comment by vijivasanthan** [comment]
[(5) 7M1: Add telecare clients on web](https://trello.com/c/eep4PPeY/989-5-7m1-add-telecare-clients-on-web)

File: app/scripts/controllers/telecareClients.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+/**
+ * Created by Henk on 10/2/2017.
```
Comment (leonjoosse) [inline app/scripts/controllers/telecareClients.js]: Are you sure about that? :)

File: app/scripts/controllers/telecareClients.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        $scope.teams = data.teams;
+        $scope.selectteams = {name:$scope.teams[0].uuid};
+
+        /**
+         * Creates a new telecare Client
+         */
+        $scope.telecareClientSubmit = function (teamName, telecareclient) {
+
+          if (typeof telecareclient == 'undefined' || !telecareclient.firstName || !telecareclient.lastName) {
+            $rootScope.notifier.error($rootScope.ui.teamup.clinetInfoFill);
```
Comment (leonjoosse) [inline app/scripts/controllers/telecareClients.js]: Minor thing: a typo in clientInfoFill

File: app/scripts/controllers/telecareClients.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+            $rootScope.notifier.error($rootScope.ui.validation.phone.notValid);
+            return;
+          }
+
+          else if ($rootScope.phoneNumberParsed.result == true) {
+            telecareclient.phone = $rootScope.phoneNumberParsed.format;
+          }
+
+          if (!telecareclient.medidoId) {
+            $rootScope.notifier.error($rootScope.ui.teamup.telecareclinetMedidoIDFill);
```
Comment (leonjoosse) [inline app/scripts/controllers/telecareClients.js]: Minor thing: a typo in clientInfoFill

File: app/scripts/locals.js [hunk 4]
```diff
@@ -3175,8 +3182,10 @@ define(
           selectTeams: 'Selecteer team(s)',
           selectTeam: 'Selecteer een team',
           clinetInfoFill: 'Vul de cliëntinformatie (naam en telefoon) in a.u.b.',
+          telecareclinetMedidoIDFill: 'Vul de MedidoID in a.u.b.',
```
Comment (leonjoosse) [inline app/scripts/locals.js]: Minor translation thing: it should be 'het MedidoID' instead of 'de'.

File: app/scripts/locals.js [hunk 5]
```diff
@@ -3074,6 +3079,8 @@ define(
           selectTeam: 'Selecteer team',
           hour: 'uur',
           clients: 'Cliënten',
+          telecareClients: 'TeleZorg Cliënten',
```
Comment (leonjoosse) [inline app/scripts/locals.js]: I think you should keep the 'z' in 'Telezorg' lowercase. Like in line 3071.

File: app/scripts/locals.js [hunk 6]
```diff
@@ -3079,7 +3079,7 @@ define(
           selectTeam: 'Selecteer team',
           hour: 'uur',
           clients: 'Cliënten',
-          telecareClients: 'TeleZorg Cliënten',
+          telecareClients: 'Telezorg Cliënten',
           telecareClientOnSucess: 'TeleZorg Cliënt succesvol toegevoegd',
```
Comment (leonjoosse) [inline app/scripts/locals.js]: Please change this one also to a lowercase 'z'

**Pull request merged**